api/tests/unit/users/test_unit_users_views.py

```diff
@@ -174,8 +174,16 @@ def test_admin_can_get_users_in_organisation(self):
             "api-v1:organisations:organisation-users-list", args=[self.organisation.pk]
         )
 
+        # add some more users to test for N+1 issues
+        for i in range(10):
+            additional_user = FFAdminUser.objects.create(
+                email=f"additional_user_{i}@org.com"
+            )
+            additional_user.add_organisation(self.organisation)
```

Comment:
Do we really need to make 10 records where 2 or 3 would be fine?

Reply:
Also a good call, I've changed to `range(2)`